src/covid_shared/cli_tools.py

```diff
@@ -58,6 +58,10 @@ def configure_logging_to_files(output_path: Path) -> None:
     click.option('-v', 'verbose',
                  count=True,
                  help='Configure logging verbosity.'),
+    click.option('-e, --email-alert',
+                 is_flag=True,
+                 default=False,
+                 help='Send email to the calling user when final job is done')
 ]
 
 
```

Comment:
Not sure the best way to rename this function in a backwards compatible way

Reply:
We could rename it with a more generic name, and then create a new function with the same original name that A) raises a deprecation warning explaining that callers should use the renamed function and B) calls the renamed function.

There's only what, like 4 packages using this? We could update each individually over the next week and then delete the deprecated function